### ISSUE TITLE
Modify Bomberman Interface to simplify restarting games

### ISSUE THREAD

**Issue by lukehsiao** [lukehsiao/BYU_Bomberman#4]
**Request**
Right now, we need a person dedicated to restarting each game after each round. This involves several keystrokes, and if the players are pressing buttons, they can interfere with the process. It would be great to modify Bomberman itself to be able to restart a round on a single button press, and to make sure the MakeyMakey's keys are mapped to ones that will not interfere with the game menus.

Perhaps, rather than selecting a level each time, we have a custom BYU map that is the only playable map. Then, it will just play that map each reset.

**Comment by lukehsiao** [comment]
One thing to also note is that the current fastest way to restart is to immediately hit `esc` after a game, exit, then restart a new game. This is much faster than waiting for the animations and such after each round.

**Comment by eaglewyng** [comment]
This looks like it might be kind of hard to implement, because we will have to recompile everything from the source. However, it can be done, and I'm looking into it.

**Comment by lukehsiao** [comment]
@eaglewyng: Yeah, it would be pretty difficult. The good news is that it's really easy to compile from source (they just have a makefile). This task is definitely very low priority. Much lower than prototyping the sandbox.
